assopy/models.py

```diff
@@ -976,8 +976,8 @@ def net_price(self):
 
 
 class RefundOrderItem(models.Model):
-    orderitem = models.ForeignKey('assopy.OrderItem')
-    refund = models.ForeignKey('assopy.Refund')
+    orderitem = models.ForeignKey('assopy.OrderItem', on_delete=models.CASCADE)
```

Comment:
same here, would replace those with classes (ie `OrderItem`) unless it causes circular. If it does – maybe add a NOTE instead.